lua/nvim-treesitter/utils.lua

```diff
@@ -64,6 +64,193 @@ function M.is_parent(dest, source)
   return false
 end
 
+function M.string_to_lines(str)
+  local t={}
+  local line_breaks = 0
+  for line in str:gmatch("(.-[\n(\r\n)\r])") do
+    table.insert(t, line)
+    line_breaks = line_breaks + 1
+  end
```

Comment:
That seems quite complicated, why not using `vim.fn.split` here ?

Reply:
This function is not used anymore. All the editing part is down by `vim.lsp.utils.apply_textedits`. As I said, I just piled up random code here. Should be fixed after the big refactoring.